src/mlpack/methods/reinforcement_learning/environment/pendulum.hpp

```diff
@@ -81,33 +81,34 @@ class Pendulum
    * In Pendulum, the action represents the torque to be applied.
    * This value is bounded in range -2.0 to 2.0 by default.
    */
-  struct Action
+  class Action
   {
+   public:
     double action[1];
```

Comment:
Since we are already changing the API, maybe we can now change it from an array to a normal double? @zoq

Reply:
Certain continuous action space envs like https://gym.openai.com/envs/LunarLanderContinuous-v2 have more than one actions. So to be consistent shouldn't we use an array for all the continuous action spaces?

Reply:
Yes good idea, wondering if we should use an array anyway even if we don't have one action?

Reply:
@nishantkr18 That makes sense to me, I guess we can close #2003 then.